routes/api.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use App\Http\Controllers\auth\LoginController;
+use App\Http\Controllers\auth\LogoutController;
+use App\Http\Controllers\auth\RegisterController;
+use Illuminate\Support\Facades\Route;
+
+Route::post('/login', LoginController::class);
```

Comment:
Please add guest middleware in route api

Reply:
okay